ipa.fc

```diff
@@ -27,3 +27,10 @@
 
 /var/run/ipa(/.*)?              gen_context(system_u:object_r:ipa_var_run_t,s0)
 
+/usr/libexec/ipa/ipa-custodia					--	gen_context(system_u:object_r:ipa_custodia_exec_t,s0)
+/usr/libexec/ipa/custodia/ipa-custodia-dmldap			--	gen_context(system_u:object_r:ipa_custodia_dmldap_exec_t,s0)
+/usr/libexec/ipa/custodia/ipa-custodia-pki-tomcat		--	gen_context(system_u:object_r:ipa_custodia_pki_tomcat_exec_t,s0)
+/usr/libexec/ipa/custodia/ipa-custodia-pki-tomcat-wrapped	--	gen_context(system_u:object_r:ipa_custodia_pki_tomcat_exec_t,s0)
+/usr/libexec/ipa/custodia/ipa-custodia-ra-agent		--	gen_context(system_u:object_r:ipa_custodia_ra_agent_exec_t,s0)
+
+/var/log/ipa-custodia.audit.log(/.*)?				--	gen_context(system_u:object_r:ipa_custodia_log_t,s0)
```

Comment:
Is this intentional? Path regex matching both file and directory, but "--" for file spec. I see it was previously there, too.

Reply:
Don't know about this one, I just moved it from ipa_custodia.fc, but it makes sence to fix it. Just not sure if it's better to exclude the directroy (to keep current labeling), or remove "--".

Reply:
@vmojzis, IMO makes sense to have specific type for the directory, too.

Reply:
Fixed, but it changes the context of /var/log/ipa-custodia.audit.log directory (assuming ipa creates one) from var_log_t to ipa_custodia_log_t.

Reply:
I removed the fix again and will create a new PR for it to ensure proper testing.